### DISCUSSION TITLE
See Laser settings on UI

### DISCUSSION THREAD

**Discussion by lottacustoms** [arkypita/LaserGRBL#1178] [Q&A]
Hi,

it would be wonderfull seeing the actual setted speed and power of the laser at the UI.

Also it would be great to save and open  "project", that means the nc. code + the original setted speed and laser power.
So if you open again you don't have to remember the settings you did successfully last time.

Best regards
Marco

**Comment by StuartB4** [comment]
`Also it would be great to save and open "project", that means the nc. code + the original setted speed and laser power.
So if you open again you don't have to remember the settings you did successfully last time.`

The speed and power settings are in the gcode. Just open the saved file and make a note of them.

**Comment by StuartB4** [comment]
I think you might have posted your reply in the wrong place, it has nothing to do with with your original request.

**Comment by lottacustoms** [comment]
Oh thank you for the hint sorry :)
